xunit.runner.visualstudio.nuspec

```diff
@@ -41,5 +41,8 @@ Supported platforms:
     <file src="build\xunit.runner.visualstudio.wpa81.targets"                                   target="build\wpa81\xunit.runner.visualstudio.targets" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.dll" target="build\wpa81\" />
     <file src="xunit.runner.visualstudio.wpa81\bin\Release\xunit.runner.visualstudio.wpa81.pri" target="build\wpa81\" />
+
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.deps.json" target="build\netstandard1.5\" />
+    <file src="xunit.runner.visualstudio.testadapter.dotnetcore\bin\Release\netstandard1.5\xunit.runner.visualstudio.testadapter.dll" target="build\netstandard1.5\" />
```

Comment:
I have several thoughts...

1. Why `netstandard1.5` as opposed to something lower like 1.0 or 1.1?
2. Does `netcoreapp1.0` really mean "an application with an entry point", or does it also apply to class libraries intended to only be consumed by .NET Core applications? (I realize that `netcore1.0` wouldn't work, since those moniker names were used for Windows Store apps).
3. Is .NET 4.6.2 actually needed here, given that we already support desktop? Is the existing desktop support not going to be extended to the "evolved" (not sure what name to use for it) multi-framework `.csproj` files?

We removed support for PCLs in our Visual Studio RockSteady plugin because we don't believe that people should be writing their unit tests as PCLs, but rather directly targeting the platform they intend to run them on. This is actually required to get device tests to run on the device with RockSteady, because of the way it works, but it's just as much a philosophical issue as much as anything else.

Given that, it seems to make sense that -- if this is truly about .NET Core -- that the unit tests should be written knowing that by actually targeting .NET Core rather than just being new-style PCLs.